### PR TITLE
chore: use prettier for code style

### PR DESCRIPTION
**Pull request by SimenB**
The code base is currently pretty inconsistent about spacing, indentation. I really love prettier, so I threw together this quick PR which uses it, and disables all styling rules from eslint itself.

(Separate PR from #611 in case you don't want this)

##### Checklist

- [x] contribution guidelines followed [here](https://github.com/nodejs/citgm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by MylesBorins** [comment]
going to let https://github.com/nodejs/citgm/pull/560 land before this to avoid conflicts

File: lib/citgm.js [hunk 1]
```diff
@@ -110,44 +125,47 @@ util.inherits(Tester, EventEmitter);
...
+      findPackageManagers,
+      tempDirectory.create,
+      grabModuleData,
+      lookup,
+      grabProject,
+      unpack,
+      packageManager.install,
+      packageManager.test
+    ],
+    err => {
```
Comment (targos) [inline lib/citgm.js]: I prefer to always have parentheses

File: .eslintrc.yml [hunk 2]
```diff
@@ -3,23 +3,14 @@
...
     eol-last: [2, always]
-    func-call-spacing: [2, never]
-    semi-spacing: 2
     strict: 2
     no-var: 2
 
-  extends: eslint:recommended
+  extends:
+      - eslint:recommended
+      - plugin:prettier/recommended
```
[the remark this answers is not in the document]
Comment (SimenB) [reply .eslintrc.yml]: I formatted the file with prettier, should be good now 🙂

**Comment by targos** [comment]
Is this way of using Prettier compatible with the VS Code extension?

**Comment by SimenB** [comment]
The VS Code eslint extension, yes. Just `eslint --fix` as usual 🙂 

I added another commit autoformatting markdown as well. Especially nice as it formats codeblocks in them. I can back out that change if you don't want it.

**Comment by targos** [comment]
I meant the VS Code Prettier extension. Does it look for the configuration you put in the package.json?

**Comment by SimenB** [comment]
Oh. Yeah, it does.

**Comment by MylesBorins** [comment]
@SimenB I've landed your other changes. Would you be able to give this one a rebase?

**Comment by SimenB** [comment]
Done!

**Comment by MylesBorins** [comment]
landed in 0b7bc4e

**Pull request closed without merging**